### DISCUSSION TITLE
BIRT Export xlsx File Merge Cell Conflict

### DISCUSSION THREAD

**Discussion by 858882847** [eclipse-birt/birt#1242] [Q&A]
[test.zip](https://github.com/eclipse/birt/files/11049597/test.zip)

**Comment by speckyspooky** [comment]
Here is your report with 3 changes:

1. That you got the error is clear for me because I wrote it in my note before. The issue is the merge footer not, if you unsplitt the row it will do correctly. The core reason is one of your columns merge before (I haven't analysed this in detail).

2. You will get a single sheet with ExcelEmitter.SingleSheet

3. I have changed your Wrapper-table and added 2 rows. Each of you tables is located in 1 row,
so your detail tables will be created from top to bottom and not side by side

![2023-03-23 20_48_21-test_version_speckyspooky xlsx - Excel](https://user-images.githubusercontent.com/41593722/227334709-ebf6f8ea-f02f-4c90-ab7e-4544b4aa720b.png)

[test_version_speckyspooky.zip](https://github.com/eclipse/birt/files/11055279/test_version_speckyspooky.zip)

**Reply by 858882847** [reply]
"I have removed the outermost layer of the table, and the three tables inside can be merged randomly. This should be a bug in xlsx. Unfortunately, if I remove the outer layer of the table, I cannot control each footer of the table when exporting it to pdf format. They will output in the last row."

**Reply by speckyspooky** [reply]
This is what hvbtup thsi week said to you, it seems not possible for your specific requirements to implements 1 report for XLSX & PDF together. Therefore his suggestion was to create 2 reports.

And if you think that is an errr then you are wolcome like contributer to enhance the XLSX-emitter.
It would be greate to get a bigger project developer team here!
